### ISSUE TITLE
Add to official base16 scheme list

### ISSUE THREAD

**Issue by augustebaum** [RRethy/base16-nvim#27]
Hi,

If it's not already in the plans, consider adding this to the [list of schemes](https://github.com/chriskempson/base16).

Nice work!

**Comment by RRethy** [comment]
I'm not sure we would be allowed since this repo doesn't actually follow some of their guidelines. The repo name and the fact we don't look at the terminals 16 colours are the main areas we break their guidelines.

**Comment by augustebaum** [comment]
Is this something you intend to work towards? Otherwise, would forking be acceptable?

**Comment by RRethy** [comment]
I'm open to PRs to add 16 terminal colour support, it would actually fix the other issue in this repo https://github.com/RRethy/nvim-base16/issues/18. I'm a bit busy right now with personal life and moving so I won't have time to implement this but near the end of november things should clear up and I can try to implement the changes. I'd be open to renaming this repo to base16-nvim to match the style guide from base 16.

**Comment by EtiamNullam** [comment]
@RRethy: I believe GitHub will still redirect from old link so changing repository name should be painless. At least that was the case for one of plugins I was using that also changed name.

More info: https://docs.github.com/en/repositories/creating-and-managing-repositories/renaming-a-repository

**Comment by RRethy** [comment]
Yup, renaming isn't a issue, the main issue is the 16 terminal colour support to make it eligible for that scheme list. This isn't super high on my priority list because most people (including myself) just use true colours.

**Comment by RRethy** [comment]
Closing due to inactivity.
